swagger.yaml

```diff
@@ -391,6 +391,69 @@ paths:
         "default":
           description: Invalid request was sent to the server.
 
+  "/users/{id}/attributes/{attributeName}":
+    put:
+      operationId: updateUserAttribute
+      summary: Update user attributes
+      description: Updates a user's attributes based on the provided data.
+      tags:
+        - attribute
+      parameters:
+        - name: id
+          in: path
+          required: true
+          description: The user attributes to be updated.
+          schema:
+            type: string
+            format: uuid
+        - name: attributeName
+          in: path  
+          required: true
+          description: The name of the attribute to be updated. Allowed attributes `isSeveraOptIn`.
+          schema:
+            type: string
+      requestBody:
+        description: updated user attributes
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/UpdateUserAttribute"
+      responses:
+        "200":
+          description: User attributes retrieved successfully
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/UpdateUserAttribute"
+        "default":
+          description: Invalid request was sent to the server.
+
+    delete:
+      operationId: removeUserAttribute
+      summary: remove user attribute
+      description: remove a specific attribute from the user's profile
+      tags:
+        - attribute
+      parameters:
+        - name: id
+          in: path
+          required: true
+          description: ID of the user whose attribute should be removed.
+          schema:
+            type: string
+        - name: attributeName
+          in: path  
+          required: true
+          description: The name of the attribute to be removed, e.g., `isSeveraOptIn`.
```

Comment:
I'm confused, isn't this keyword not attribute? Attribute related to keycloak and is the severaUserId stored there. 

I think the descriptions at least should be updated for the PUT and the delete to make it clear we optIn/ optOut which involves adding/ removing keycloak attributes and severa keywords